README.md

```diff
@@ -1,6 +1,7 @@
 # Skylib
 
 [![Build Status](https://travis-ci.org/bazelbuild/bazel-skylib.svg?branch=master)](https://travis-ci.org/bazelbuild/bazel-skylib)
+[![Build status](https://badge.buildkite.com/921dc61e2d3a350ec40efb291914360c0bfa9b6196fa357420.svg)](https://buildkite.com/bazel/bazel-skylib-postsubmit)
```

Comment:
https://buildkite.com/bazel/bazel-skylib-postsubmit seems to come back as page not found, expected?

Reply:
Yep, please see https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md ... one needs to be logged in for now :(

Reply:
Looks like one needs to be logged in an have some extra permissions, as I'm logged in, but that link doesn't work.  Likewise, https://buildkite.com/bazel/ comes back with:

>You don’t have permission to view or create any pipelines because you’ve not been assigned to a team.
>
>Please contact one of your team maintainers or organization administrators for help.